drools-wb-screens/drools-wb-enum-editor/drools-wb-enum-editor-backend/src/test/java/org/drools/workbench/screens/enums/backend/server/indexing/IndexEnumEntriesTest.java

```diff
@@ -33,6 +33,7 @@
 import org.uberfire.ext.metadata.io.KObjectUtil;
 import org.uberfire.java.nio.file.Path;
 
+@org.junit.Ignore
```

Comment:
Why is this now ```@Ignore```'ed?

Reply:
Oops :-( I accidentally committed this. It's fixed now.